config/config.exs

```diff
@@ -101,6 +101,14 @@ config :esbuild,
          --outdir=../priv/static/assets --external:/fonts/* --external:/images/*),
     cd: Path.expand("../assets", __DIR__),
     env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
+  ],
+  monaco: [
```

Comment:
Here we instruct `esbuild` to copy the `monaco files` to our assets directory

Reply:
In dev, this adds about 50mb of _stufff_ to `static/assets`. I presume it's the same in prod. 

I don't _think_ this matters? Most of it will sit on the filesystem and never be used. The client is downloading exactly the same files, just from a different location